lib/editor/util/__tests__/__snapshots__/map.js.snap

```diff
@@ -583,46 +448,22 @@ Object {
   "coordinates": Array [
     Array [
       Array [
-        -77.922203,
-        42.086407,
-      ],
-      Array [
-        -77.922196,
-        42.086498,
-      ],
-      Array [
-        -77.922173,
-        42.086498,
+        -77.9230535030365,
+        42.096176462786296,
```

Comment:
This snapshot for `should update a point in the middle of the control points` seems really off again.

Before:

<img width="235" alt="Screen Shot 2019-04-03 at 4 41 44 PM" src="https://user-images.githubusercontent.com/3112493/55520076-8406ca80-562f-11e9-9ce4-0e33bf3048c2.png">

After:

<img width="239" alt="Screen Shot 2019-04-03 at 4 41 35 PM" src="https://user-images.githubusercontent.com/3112493/55520087-8a954200-562f-11e9-9892-c9526a403b24.png">